KsApi/Service.swift

```diff
@@ -460,7 +460,7 @@ public struct Service: ServiceType {
     return request(.updateUserSelf(user))
   }
 
-  private func decodeModel<M: Decodable>(_ json: Any) ->
+  private func decodeModel<M: Argo.Decodable>(_ json: Any) ->
```

Comment:
Most of this diff was a quick find/replace to address an Argo conflict with Swift's new Codable stuff! (See https://github.com/thoughtbot/Argo/issues/466)